docs/get_started/license.rst

```diff
@@ -2,9 +2,29 @@
 License
 =======
 
-GNU GENERAL PUBLIC LICENSE, Version 3, 29 June 2007
-------------
+Current License
```

Comment:
Should we add some info about the current license here? Or at least give a link?

Reply:
It is the text box right below. I'll add text for clarification

Reply:
Ahh I see now